packages/manager/apps/hub/src/catalog/catalog.scss

```diff
@@ -1,4 +1,11 @@
 hub-catalog {
```

Comment:
Maybe the component `<hub-catalog>` could be centered by default (instead of targeting direct `div` elements).

Is that make sense?

Reply:
i tried to apply the style directly on the component but it didn't work :/